### ISSUE TITLE
JeaSessionConfiguration failing consistency check in Azure Automation State Config

### ISSUE THREAD

**Issue by Weatwagon** [ChrisLGardner/JeaDsc#25]
The initial consistency check in Azure Automation State Config comes back as compliant. The Jea endpoint functions as expect but it is finding issue with the JeaSessionConfiguration during subsequent consistency checks. These checks come back as failed. 

The error in Azure is also same as Output of Test-DscConfiguration ran on same node
```
The term 'Convert-StringToArrayOfObject' is not recognized as the name of a cmdlet, function, script file, or operable
program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
    + CategoryInfo          : ObjectNotFound: (Convert-StringToArrayOfObject:) [], CimException
    + FullyQualifiedErrorId : CommandNotFoundException
    + PSComputerName        : localhost

The PowerShell DSC resource '[JeaSessionConfiguration]JeaEndpoint' with SourceInfo '::13::5::JeaSessionConfiguration'
threw one or more non-terminating errors while running the Test functionality. These errors are logged to the ETW
channel called Microsoft-Windows-DSC/Operational. Refer to this channel for more details.
    + CategoryInfo          : InvalidOperation: (root/Microsoft/...gurationManager:String) [], CimException
    + FullyQualifiedErrorId : NonTerminatingErrorFromProvider
    + PSComputerName        : localhost
```
my dsc config

```
Configuration JEAConfig
{
    Import-DscResource -Module JeaDsc

    
    JeaRoleCapabilities TechnicianRoleCapability
    {
        Path = 'C:\Program Files\WindowsPowerShell\Modules\JeaDsc\RoleCapabilities\Technician.psrc'
        VisibleCmdlets = 'Get-WinEvent','Get-Service'                
    }

    JeaSessionConfiguration JeaEndpoint
    {
        EndpointName = "JeaEndpoint"
        RoleDefinitions = "@{ 'Contoso\Infrastructure Support' = @{ RoleCapabilities = 'Technician' }; }"
        TranscriptDirectory = 'C:\ProgramData\JeaEndpoint\Transcripts'        
        DependsOn = '[JeaRoleCapabilities]TechnicianRoleCapability'
    }    
}
```

**Comment by ChrisLGardner** [comment]
I've spotted the issue and will have a new version out in the next few hours, should be 0.6.3.

**Comment by ChrisLGardner** [comment]
This should be fixed now in 0.6.3, can you give it a try and let me know if you're having any more issues.

**Comment by Weatwagon** [comment]
New error but I'm sure its the same fix you did for 'Convert-StringToArrayOfObject'

```
The term 'Convert-StringToArrayOfHashtable' is not recognized as the name of a cmdlet, function, script file, or
operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try
again.
    + CategoryInfo          : ObjectNotFound: (Convert-StringToArrayOfHashtable:) [], CimException
    + FullyQualifiedErrorId : CommandNotFoundException
    + PSComputerName        : localhost

The PowerShell DSC resource '[JeaSessionConfiguration]JeaEndpoint' with SourceInfo '::13::5::JeaSessionConfiguration'
threw one or more non-terminating errors while running the Test functionality. These errors are logged to the ETW
channel called Microsoft-Windows-DSC/Operational. Refer to this channel for more details.
    + CategoryInfo          : InvalidOperation: (root/Microsoft/...gurationManager:String) [], CimException
    + FullyQualifiedErrorId : NonTerminatingErrorFromProvider
    + PSComputerName        : localhost
```

**Comment by ChrisLGardner** [comment]
Bugger, I'll get that fixed. I also need to find some time to add some integration tests into the session config side of this so that I can catch these earlier.

**Comment by ChrisLGardner** [comment]
And that should now be fixed in 0.6.4

**Comment by Weatwagon** [comment]
That did the trick. Showing compliant now. Thank you for the help.
